README.md

```diff
@@ -82,15 +82,43 @@ InventoryItem(name='bar', price=12.7, stock=9)
 True
 ```
 
-### Converting to dictionary
+### Converting to dict
 
-Passing an `ItemAdapter` to the `dict` built-in will create a new dictionary:
+The simple way: passing an `ItemAdapter` to the `dict` built-in will create a new dictionary:
 
 ```python
 >>> dict(adapter)
 {'name': 'bar', 'price': 12.7, 'stock': 9}
 ```
 
+However, this doesn't work for nested items. In those cases, you could use the
+`ItemAdapter.asdict` method. Consider the following example:
+
+```python
+from dataclasses import dataclass
+
+@dataclass
+class Price:
+    value: int
+    currency: str
+
+@dataclass
+class Product:
+    name: str
+    price: Price
+```
+
+```python
+>>> item = Product("Stuff", Price(50, "UYU"))
+>>> adapter = ItemAdapter(item)
+
+>>> dict(adapter)  # not recursive
+{'name': 'Stuff', 'price': Price(value=50, currency='UYU')}
+
+>>> adapter.asdict()  # recursive
+{'name': 'Stuff', 'price': {'currency': 'UYU', 'value': 50}}
+```
+
```

Comment:
Instead of covering `dict(object)` first as “the simple way”, and `asdict()` later, what about covering first `asdict`, and then covering `dict(object)` later as “the way to avoid recursive conversion”? (assuming recursive conversion will be desired sometimes but less frequently)